add note on de-duping styled-components (#238)
BBC-archive/psammead: README.md

[See documentation on the Styled Components site](https://www.styled-components.com/docs/tooling#babel-plugin)

> **NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups.

Author: ```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups, although there is an [issue with symlinking `jest-styled-components` mangling snapshots](https://github.com/styled-components/jest-styled-components/issues/218).
```

> **NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicated `styled-components` somewhere in your dependencies, which is not compatible with `styled-components` which acts as a singleton. You can run [npm dedupe](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) or [write your own bash script](https://github.com/BBC-News/psammead/blob/9f30156d5a602b0dbeb427c29568022284746fee/package.json#L9) to resolve the duplication.

Author: ```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups.
```

> **NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.

Reviewer: ```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups such as Psammead's. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```

> **NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages, which conflicts with its singleton structure. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups, although there is an [issue with symlinking `jest-styled-components` mangling snapshots](https://github.com/styled-components/jest-styled-components/issues/218).

Author: ```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups (although there is an [open issue with symlinking `jest-styled-components`](https://github.com/styled-components/jest-styled-components/issues/218)). Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```

> **NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups (although there is an [open issue with symlinking `jest-styled-components`](https://github.com/styled-components/jest-styled-components/issues/218)). Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.

Author: ```suggestion
**NOTE**: if you run into issues with CSS not being applied to your components, it is likely that there is a duplicate `styled-components` dependency somewhere in your packages. You can try running [`npm dedupe`](https://www.styled-components.com/docs/faqs#duplicated-module-in-node_modules) in most cases, or [`lerna bootstrap --hoist`](https://www.styled-components.com/docs/faqs#usage-with-lerna) in monorepo setups. Failing that, make sure your application's `styled-components` dependency is the same version as that in Psammead.
```
